reference/yaml-format-reference.md

```diff
@@ -1,88 +1,117 @@
+## Where OpenRewrite YAML files can exist
 
-{% hint style="info" %}
-Within a YAML document recipe and style names are always fully qualified.
-{% endhint %}
+There are two places where you can define an OpenRewrite YAML file:
 
-{% hint style="warning" %}
-Don't place your custom recipes into the org.openrewrite namespace. We recommend using the same "reverse domain name" naming scheme used in Java packages.
-{% endhint %}
+1. Within the `rewrite.yml` file of a project that applies rewrite recipes via the [rewrite-gradle-plugin](gradle-plugin-configuration.md) or [rewrite-maven-plugin](rewrite-maven-plugin.md)
+2. Inside the `META-INF/rewrite` folder of a jar (such as in the [rewrite-testing-frameworks](https://github.com/openrewrite/rewrite-testing-frameworks/tree/master/src/main/resources/META-INF/rewrite))
+
+If you define a recipe or style in the `rewrite.yml` file, they _will not_ be included in the jars published from your project.
+
+If you want to redistribute a recipe or a style and apply them to other projects, you'll need to create them inside of the `META-INF/rewrite` folder of a jar.
+
+## Best practices
+
+Please keep these conventions in mind when you're creating OpenRewrite YAML files:
+
+* A file may contain any number of recipes and styles, separated by `---`.
+* Within a file, recipe and style names must be fully qualified.
+* Custom recipes should not be placed into the `org.openrewrite` namespaced. Instead, they should follow the same [reverse domain name notation](https://en.wikipedia.org/wiki/Reverse_domain_name_notation) used in Java packages. 
 
 ## Recipes
 
-The basic format of a recipe defined in YAML is:
+### Format
 
-```yaml
----
-type: specs.openrewrite.org/v1beta/recipe
-name: com.yourorg.RecipeA     # A fully qualified, unique name for this recipe
-displayName: Recipe A         # A human-readable name for this recipe
-description: Applies Recipe B # A human-readable description for this recipe
-recipeList:                   # the list of recipes which comprise this recipe
-  - com.yourorg.RecipeB:      # A fully qualified name for a recipe defined elsewhere
-      exampleConfig: foo      # A configurable parameter of RecipeB
-```
+| Key                         | Type             | Description                                                             |
+|-----------------------------|------------------|-------------------------------------------------------------------------|
+| type                        | const            | A constant:  `specs.openrewrite.org/v1beta/recipe`                      |
+| name                        | string           | A fully qualified, unique name for this recipe                          |
+| displayName                 | string           | A human-readable name for this recipe                                   |
+| description                 | string           | A human-readable description for this recipe                            |
```

Comment:
Maybe include that displayNames do not end in a dot, such as a title,
whereas descriptions do end in a dot, such as a sentence.